EventFilter/CTPPSRawToDigi/python/ctppsDigiToRaw_cfi.py

```diff
@@ -0,0 +1,27 @@
+import FWCore.ParameterSet.Config as cms
+from EventFilter.RawDataCollector.rawDataCollector_cfi import *
+
+#from EventFilter.CTPPSRawToDigi.ctppsPixelRawData_cfi import ctppsPixelRawData
+#ctppsPixelRawData.InputLabel = cms.InputTag("RPixDetDigitizer")
+
+ctppsPixelRawData = cms.EDProducer("CTPPSPixelDigiToRaw",
+    InputLabel = cms.InputTag("RPixDetDigitizer"),
+    mappingLabel = cms.string("RPix") 
+)
+
+ctppsTotemRawData = cms.EDProducer("CTPPSTotemDigiToRaw",
+    InputLabel = cms.InputTag("RPSiDetDigitizer")
+)
+
+ctppsRawData = cms.Sequence(ctppsTotemRawData*ctppsPixelRawData)
```

Comment:
it seems like the default should be an empty sequence and a non-empty should be defind with ctpps_2016